GameDataParser/Parsers/SkillParser.cs

```diff
@@ -88,55 +102,38 @@ protected override List<SkillMetadata> Parse()
                     XmlNodeList jobs = document.SelectNodes("/ms2/job");
                     foreach (XmlNode job in jobs)
                     {
-                        if (job.Attributes["feature"] != null) // Getting attribute that just have "feature"
+                        // Grabs all the skills and them the jobCode.
+                        XmlNodeList skills = job.SelectNodes("skills/skill");
+                        int jobCode = int.Parse(job.Attributes["code"].Value);
+                        foreach (XmlNode skill in skills)
                         {
-                            string feature = job.Attributes["feature"].Value;
-                            if (feature == "JobChange_02") // Getting JobChange_02 skillList for now until better handle Awakening system.
```

Comment:
Why was this removed?

Reply:
What this was use for is to set just the `feature =JobChange_02` skills in the job.xml, and with the new changes we are setting all skills and subskills to their job code, including beginner.